### ISSUE TITLE
[FEATURE] Allow no closing `djlint:on` comment

### ISSUE THREAD

**Issue by twolfson** [Riverside-Healthcare/djLint#649]
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
    - Sorry if I missed it =/ I tried "closing", "closing djlint" and "closing djlint:on"
- [x] I've read the [docs](https://djlint.com)
    - Read through https://www.djlint.com/docs/ignoring-code/

## Feature Request

Loving the idea behind this library! I'm coming from being very used to `prettier` and `black` to cut out dev time just indenting =)

During my initial testing and setup though, I ran into a snag with how `djlint:off` is handled.

**Setup:**

index.html:

```django
{# djlint:off #}
<div style="background: linen; height: 100vh">
    Hello
</div>
```

**Expected behavior:**

```bash
djlint index.html
# Should see success, no error code
```

**Actual behavior:**

```bash
djlint index.html
#####
index.html
────────────────────────────────
H021 2:0 Inline styles should be avoided. <div style=

Linted 1 file, found 1 error.
```

To remove the issue, I need to provide a `djlint:on` comment it seems, which was not obvious from the docs =/

(I tried older versions as part of troubleshooting because I was so confused)

```
{# djlint:off #}
<div style="background: linen; height: 100vh">
    {# djlint:on #}
    Hello
</div>
```

Could we consider adding support for not specifying a closing `djlint:on` for a file?

**Comment by christopherpickering** [comment]
Thanks, I can check that out, it's more of a bug than feature probably :)

**Comment by twolfson** [comment]
Geez, y'all are speedy!! Thank you!! =D

**Comment by twolfson** [comment]
Ah, sadly this only works when there are no error codes specified x_x Will open a new ticket for that nuance =)

**Comment by christopherpickering** [comment]
Thanks, sorry I didn't think of that one, the linter logic is a bit split out. Thanks for  opening another issue.
